util/git/creds.go

```diff
@@ -245,6 +266,13 @@ func NewSSHCreds(sshPrivateKey string, caPath string, insecureIgnoreHostKey bool
 	return SSHCreds{sshPrivateKey, caPath, insecureIgnoreHostKey, store, proxy, noProxy}
 }
 
+// GetUserInfo returns empty strings for user info.
+// TODO: Implement this method to return the username and email address for the credentials, if they're available.
```

Comment:
Create issues if this is still TODO and it could be implemented in another PR

Reply:
https://github.com/argoproj/argo-cd/issues/20845